test/core/domainerTests.ts

```diff
@@ -150,15 +155,16 @@ describe("Domainer", () => {
   });
 
   it("include(n) works on dates", () => {
-    var a = new Date(2000, 5, 4);
-    var b = new Date(2000, 5, 5);
-    var c = new Date(2000, 5, 6);
-    var d = new Date(2003, 0, 1);
-    domainer.addIncludedValue(b);
+    var includedDate = new Date(2000, 5, 5);
+    var startDate = new Date(2000, 5, 6);
+    var endDate = new Date(2003, 0, 1);
     var timeScale = new Plottable.Scales.Time();
-    timeScale._updateExtent("1", "x", [c, d]);
+    var provider: Plottable.Scales.ExtentProvider<Date> = (scale: Plottable.Scale<Date, number>) => [[startDate, endDate]];
+    timeScale.addExtentProvider(provider);
```

Comment:
Hey, if you just plug in the provider inline-style since you don't use the variable anywhere